website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'React Native 0.71: TypeScript by Default, Flexbox Gap, and more...'
+authors: [lunaleaps, mattcarrollcode, cortinico, kelset]
+tags: [announcement, release]
+date: 2023-01-12
+---
+
+Today we’re releasing React Native version 0.71. This is a feature-packed release which includes TypeScript by default, Flexbox gap support, new web-inspired props, developer experience improvements, and improvement to the New Architecture experimental experience.
+
+## TypeScript by default
+
+In this release, we’re investing in the TypeScript experience of React Native.
+
+Starting with 0.71, when you create a new React Native app via the React Native CLI you'll get a TypeScript app by default. The new project is already set up with a `tsconfig.json` so out of the box your IDE will help you write typed code right away.
+
+We’re also offering built-in TypeScript declarations directly from the `react-native` package. This means you won’t need `@types/react-native` any longer.
```

Comment:
Do you want to throw in "more accurate" or something like that?

> offering built-in, more accurate TypeScript declarations...

Because the key point isn't just that they need one less npm package, right? It's that the quality of the definitions should be better now and going forward.